### PR TITLE
Ref task #9294 New/Edit Order Fragment - Fetch data v2

### PR DESCRIPTION
**Pull request by trunghq-1746**
## Issue Link :link:

- https://edu-redmine.sun-asterisk.vn/issues/9294

## Goals :soccer:

- Fetch ward data
- Fetch shipping fee

### CODE REVIEW

**Comment by trunghq-1746** [comment]
ready

File: app/src/main/java/com/trunghoang/orderhub/model/GHNApiRequest.kt [hunk 1]
```diff
@@ -1,7 +1,42 @@
...
+        const val DEFAULT_WEIGHT = 500
+    }
+
     class Districts(
         val token: String
     )
+
+    class Wards(
+        val token: String,
+        val DistrictID: Int? = null
```
Comment (DoanVanToan) [inline app/src/main/java/com/trunghoang/orderhub/model/GHNApiRequest.kt]: check convention class này

File: app/src/main/java/com/trunghoang/orderhub/model/GHNApiResponse.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    class Wards(
+        val Wards: List<Ward>
+    )
+
+    class Fee(
+        val CalculatedFee: Int
+    )
+
+    class Service(
+        val ExpectedDeliveryTime: String,
```
Comment (DoanVanToan) [inline app/src/main/java/com/trunghoang/orderhub/model/GHNApiResponse.kt]: tuơng tự class này

**Comment by trunghq-1746** [comment]
ready

**Pull request merged**